cmd/agentbasedinstaller/import.go

```diff
@@ -14,8 +15,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func ImportCluster(ctx context.Context, log *log.Logger, bmInventory *client.AssistedInstall,
-	clusterID strfmt.UUID, clusterName string, clusterAPIVIPDNSName string, clusterConfigDir string) (*models.Cluster, error) {
+const (
+	workerIgnitionEndpointFilename = "worker-ignition-endpoint.json"
+	importClusterConfigFilename    = "import-cluster-config.json"
+)
+
+func ImportCluster(fsys fs.FS, ctx context.Context, log *log.Logger, bmInventory *client.AssistedInstall,
```

Comment:
w.r.t. above comment, append `fsys fs.FS` at the end of existing function params